APITest/APITest/Controllers/EmployeeController.cs

```diff
@@ -1,23 +1,77 @@
-﻿using RestSharp;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
+using RestSharp;
+using APITest.Constants;
 
 namespace APITest.Controllers
 {
-    public class EmployeeController : BaseController
+    public class EmployeeController
     {
-        private const string GetEmployeeUrl = "/employees";
-        private const string GetEmployeeByIdUrl = "/employees/{0}";
```

Comment:
Why do you enter here constants? You have a special class ConfigConstants for this. You should use their data.

Reply:
This is old and not unactual code